.storybook/main.js

```diff
@@ -18,5 +33,6 @@ module.exports = {
     },
   ],
   // Use glob to locate the stories, because it ignores our circular dependencies:
+  // Remove .. ?
   stories: glob.sync("../**/*.stories.@(js|jsx|ts|tsx)", { cwd: __dirname }),
```

Comment:
FWIW, I do think we'll still need this.
In my testing, I put an invalid `stories/INVALID.stories.ts` file.  The OOM error still happened, and it never even complained about the invalid syntax in the file.  So this tells me that it was stalling when "finding all stories" recursively, before it even checked what `imports` they had.

Reply:
Sounds good -- I've removed the `// Remove` comment